packages/forms-web-app/src/lib/views.js

```diff
@@ -55,6 +55,8 @@ const VIEW = {
     ANY_OF_FOLLOWING: 'before-you-start/any-of-following',
     LOCAL_PLANNING_DEPARTMENT: 'before-you-start/local-planning-department',
     TYPE_OF_PLANNING_APPLICATION: 'before-you-start/type-of-planning-application',
+    ENFORCEMENT_NOTICE: 'before-you-start/enforcement-notice',
```

Comment:
This is fine, but we have a linitng error (space) on line 45

Reply:
Resolved